src/Home.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
```

Comment:
Home.jsx probably doesn't belong in the root of src/ anymore, now that it's just a page.  
Maybe we should create a pages/ or views/ directory to contain these